train.py

```diff
@@ -279,12 +280,13 @@ def fit(self, epochs):
         Returns:
             history of training and validation losses for this training session
         """
-        print_interval = epochs // 10 + int(epochs < 10)
         train_losses = []
         val_losses = []
         start = time.time()
 
         print("Beginning training...")
+        print(time.strftime("%Y-%m-%d %H:%M"))
+        torch.set_float32_matmul_precision("high") # this speeds up traning
 
```

Comment:
Add the following changes to use the compiled model while training
```suggestion
model = torch.compile(self.model)
```
And then replace all the instances of `self.model` in the `fit` method with `model`

Reply:
Will do

Reply:
Done